janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -292,6 +293,18 @@ public boolean apply(@Nullable String s) {
                     "default", JanusGraphDefaultSchemaMaker.INSTANCE,
                     "tp3", Tp3DefaultSchemaMaker.INSTANCE);
 
+    public static final ConfigOption<String> MAPPING_HANDLING = new ConfigOption<>(SCHEMA_NS, "mapping",
+        "Configures the behavior of the schema handling. If set to 'none', mapping constraints are not applied. " +
```

Comment:
Configures the schema mode to be used by this graph. If set to 'none', no constraints are applied. If set to 'constrained', constraints on properties per label and possible connections are enforced.